gulpfile.js

```diff
@@ -53,10 +53,15 @@ function getPlatforms(includeChromeOs) {
             platforms.push('osx64');
 
             break;
-        case 'linux':
+        case 'linux64':
```

Comment:
This won't work, since `os.platform()` will only ever return `linux`.
You'd need to check `os.arch()` (https://nodejs.org/api/os.html#os_os_arch) to find the processor architecture.

But, as discussed, the official release will only contain the x64 build for linux, so any call to `gulp` without an explicit argument should build that version. (I.e. requiring users to specify `--linux32` on the command line to build for linux32.)